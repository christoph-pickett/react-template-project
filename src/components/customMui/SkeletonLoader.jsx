import { Skeleton } from "@mui/material";

export const SkeletonLoader = ({ height }) => {
  return (
    <Skeleton
      variant="rectangular"
      width={"inherit"}
      height={height ? height : "500px"}
    />
  );
};
