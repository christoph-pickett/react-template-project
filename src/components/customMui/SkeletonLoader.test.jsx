import { render } from "@testing-library/react";
import { SkeletonLoader } from "./SkeletonLoader";

test("renders SkeletonLoader component correctly", () => {
  const { container } = render(<SkeletonLoader />);
  expect(container).toMatchSnapshot();
});
