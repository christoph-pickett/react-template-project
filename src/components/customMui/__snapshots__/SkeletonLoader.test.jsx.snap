// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SkeletonLoader component correctly 1`] = `
<div>
  <span
    class="MuiSkeleton-root MuiSkeleton-rectangular MuiSkeleton-pulse css-153n0d3-MuiSkeleton-root"
    style="width: inherit; height: 500px;"
  />
</div>
`;
